tests/testthat/test-create_ipctwg_table.R

```diff
@@ -0,0 +1,182 @@
+test_that("that the results are correctly displayed", {
```

Comment:
If I add any additional value in the _values(fclc_matrix_values/fcs_cat_values..) the then function runs smoothly. Perhaps it should show a warning, and/or restrict `users` not to put more than 3/5 values?

```
actual_output <- create_ipctwg_table(.results = presentresults_resultstable,
                                     fclc_matrix_var = "fcls_cat",
                                     fclc_matrix_values = c("xxxxxxxxxxxx","phase_1", "phase_2", "phase_3", "phase_4", "phase_5"),
                                     fc_matrix_var = "fcm_cat",
                                     fc_matrix_values = c("phase_1", "phase_2", "phase_3", "phase_4", "phase_5"),
                                     with_fclc = T,
                                     fcs_cat_values = c("low", "medium", "high"),
                                     rcsi_cat_values = c("low", "medium", "high"),
                                     lcsi_cat_var = "lcs_cat",
                                     lcsi_cat_values = c("none", "stress", "emergency", "crisis"),
                                     hhs_cat_var = "hhs_cat",
                                     hhs_cat_values = c("none", "slight", "moderate", "severe", "very_severe"),
                                     lcsi_set = c("liv_stress_lcsi_1",
                                                  "liv_stress_lcsi_2",
                                                  "liv_stress_lcsi_3",
                                                  "liv_stress_lcsi_4",
                                                  "liv_crisis_lcsi_1",
                                                  "liv_crisis_lcsi_2",
                                                  "liv_crisis_lcsi_3",
                                                  "liv_emerg_lcsi_1",
                                                  "liv_emerg_lcsi_2",
                                                  "liv_emerg_lcsi_3"))

```

Reply:
Also if i remove values for `lcsi_set ` then no warninng is showing. Perhaps we can add something?

Reply:
added verify_numbers_values and the  test_that("if number of expected values is different it will show a warning", {